packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -154,6 +189,10 @@ function ImportModal({
     startImport();
   }, [startImport]);
 
+  const handleSkipCSVAnalyze = useCallback(() => {
+    skipCSVAnalyze();
+  }, [skipCSVAnalyze]);
```

Comment:
Just a note that when written like that, useCallback adds no memoization value here, but an additional function call, which is sort of unnecessary. I'm guessing it just follows the pattern in the component, so maybe something to keep in mind for the future

Reply:
yeah I just copied the stuff around it. But good point.

Reply:
Cleaned that up now.